library/core/registry/src/main/java/org/quiltmc/qsl/registry/mixin/SimpleRegistryMixin.java

```diff
@@ -107,7 +128,7 @@ protected SimpleRegistryMixin(RegistryKey<? extends Registry<V>> key, Lifecycle
 			at = @At("RETURN")
 	)
 	private void quilt$invokeEntryAddEvent(int rawId, RegistryKey<V> key, V entry, Lifecycle lifecycle,
-			CallbackInfoReturnable<Holder<V>> cir) {
+										   CallbackInfoReturnable<Holder<V>> cir) {
```

Comment:
Beware of the autoformatter aligning stuff!
This is not wanted as it often tries to use spaces instead of tabs and we're tabs only.

Reply:
ah, I'll update my IJ settings then